### DISCUSSION TITLE
These errors occured while restoring the configuration...

### DISCUSSION THREAD

**Discussion by neosmith20** [EvanMulawski/FanControl.CorsairLink#75] [General]
So, there are times when Fan Control starts up and it throws this error. It can't find any of my fans or sensors from Corsair. It seems to be random, I'll restart the program, or even the pc, and it's a 50/50 chance that it will work. iCue is disabled and I'm running a Commander Core XT. I had everything setup, fan/sensor wise, but every now and then it wont see anything. Not sure what I need to do so this stops happening and it works every time. If I hit "Refresh" there are times that it will detect everything and go on just fine, and then there are times when I hit "Refresh" it keeps throwing this error. I'm also running the latest version of Fan Control (V165)

![Corsair Errors](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/49753731/be7a84e1-4669-4539-9ff6-45e8eaaea371)
[log.txt](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12270425/log.txt)

**Comment by EvanMulawski** [comment]
Hi @neosmith20, the log indicates the plugin can successfully open the device and send a command to it but is waiting for a valid response that is never received. This is usually an indication that incompatible software is communicating with the device.

Ensure the "Corsair Service" Windows service is not running and only [compatible](https://github.com/EvanMulawski/FanControl.CorsairLink#interoperability) software is running alongside Fan Control. Additionally, consider [updating](https://github.com/EvanMulawski/FanControl.CorsairLink/releases) the plugin to v1.3.0.

**Reply by neosmith20** [reply]
I updated the plugin and followed the directions to install everything it needed. I also made sure that corsair service was disabled, and it was as I changed it to disabled awhile back. It kept throwing the same error and after looking and looking i found that the icue software was still open and hadn't fully quit. after I killed that process, I reopened "Fan Control" and everything came back working as it should.

Thanks for the help! 👍
